src/jit/codegenlinear.h

```diff
@@ -135,6 +132,11 @@ void genFMAIntrinsic(GenTreeHWIntrinsic* node);
 void genLZCNTIntrinsic(GenTreeHWIntrinsic* node);
 void genPCLMULQDQIntrinsic(GenTreeHWIntrinsic* node);
 void genPOPCNTIntrinsic(GenTreeHWIntrinsic* node);
+template <typename HWIntrinsicSwitchCaseBody>
+void genJumpTableFallback(regNumber                 nonConstImmReg,
```

Comment:
Should this be called `genHWIntrinsic_JumpTableFallback` (since it is specific to HWIntrinsic codegen)?

Reply:
>  it is specific to HWIntrinsic 

Both okay to me. We do have other "JumpTableFallback" now.

Reply:
Yes, but we don't want future code to think it is safe to use as general purpose (if it isn't, and if it is, the typename should be changed).

Reply:
Okay, will change.